net-libs/libwebsockets/libwebsockets-1.7.4.ebuild

```diff
@@ -0,0 +1,27 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+inherit cmake-utils
+
+DESCRIPTION="canonical libwebsockets.org websocket library"
+HOMEPAGE="https://libwebsockets.org/"
+SRC_URI="https://github.com/warmcat/libwebsockets/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="LGPL2"
```

Comment:
The contents LICENSE must be a valid file in the repo (eg. /usr/portage/licenses), and repoman will warn about that. It looks like the correct LICENSE for this package is LGPL-2.1.